tools/update-docs-screenshots.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 
-export SHOT="$(dirname "$(readlink -f "$0")")/screenshot-xodball"
+export SHOT="$(cd "$(dirname "$0")"; pwd)/screenshot-xodball"
```

Comment:
OK, but can we have a little comment describing what the hell is going here?

Reply:
Relative path to `tools/update-docs-screenshots.sh` ($0) -> relative path of directory `tools` -> absolute path to directory `tools` -> absolute path to `tools/screenshot-xodball`.

Reply:
A good explanation for the comment I’m asking for :smiley: